Client/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -102,38 +102,63 @@ bool CScriptDebugging::SetLogfile(const char* szFilename, unsigned int uiLevel)
         // round 37.5 to 38 because we can't have half a message
         // 8 * 256 bytes = 6004B
         // round 6004 up to the nearest divisible by 1024 = 6144
-        // we have our buffer size.
-        setvbuf(pFile, NULL, _IOFBF, 6144);
+        setvbuf(pFile, nullptr, _IOFBF, 6144);
 
         // Set the new pointer and level and return true
         m_uiLogFileLevel = uiLevel;
         m_pLogFile = pFile;
 
         // Create a timer
-        ::CreateTimerQueueTimer(&m_flushTimerHandle, NULL, TimerProc, NULL, 50, 50, WT_EXECUTEINTIMERTHREAD);
+        ::CreateTimerQueueTimer(&m_flushTimerHandle, nullptr, TimerProc, nullptr, 50, 50, WT_EXECUTEINTIMERTHREAD);
         return true;
     }
 
     return false;
 }
 
+enum DebugScriptLevels : std::uint8_t
```

Comment:
This is already defined in Shared/mods/deathmatch/logic/CScriptDebugging.cpp. Please make use of that one